test/unit/datetime.result

```diff
@@ -1,362 +1,504 @@
-1..4
-    1..355
+1..5
```

Comment:
Now that the patch removing unit/datetime.result is merged you may rebase your patchset on current master and remove this huge diff chunk altogether.

Reply:
Done.